collections/St_Johns_College/SW_V1_St_Johns_College_MS_006.xml

```diff
@@ -0,0 +1,233 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://raw.githubusercontent.com/msDesc/consolidated-tei-schema/master/msdesc.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="https://raw.githubusercontent.com/msDesc/consolidated-tei-schema/master/msdesc.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xmlns:tei="http://www.tei-c.org/ns/1.0"
+   xml:id="manuscript_12006">
+   <teiHeader xml:lang="en">
+      <fileDesc>
+         <titleStmt>
+            <title>St John's College MS. 6</title>
+            <title type="collection">St John's College MSS.</title>
```

Comment:
please could you change all of these to read "St John's College MSS. (in progress)" - we'll change it back at the end of the project